### ISSUE TITLE
Should this still be working?

### ISSUE THREAD

**Issue by Dids** [nbering/terraform-provider-ansible#28]
After copying `terraform-provider-ansible_v1.0.3` to `.terraform/plugins/drawin_amd64`, Terraform is still complaining about not being able to find the provider:

```
Provider "ansible" not available for installation.

A provider named "ansible" could not be found in the Terraform Registry.

This may result from mistyping the provider name, or the given provider may
be a third-party provider that cannot be installed automatically.

In the latter case, the plugin must be installed manually by locating and
downloading a suitable distribution package and placing the plugin's executable
file in the following directory:
    terraform.d/plugins/darwin_amd64

Terraform detects necessary plugins by inspecting the configuration and state.
To view the provider versions requested by each module, run
"terraform providers".


Error: no provider exists with the given name
```

**Comment by Dids** [comment]
Figured out the issue. This breaks when using `terraform init -upgrade`, but works fine when omitting `-upgrade`.

Since I'd like to be able to upgrade my other plugins, is there a way to skip checking this plugin for updates?

**Comment by nbering** [comment]
Interesting to note. I'll look into this if I get a chance. Might be worth a footnote in the README.

**Comment by rgl** [comment]
I think this provider must be published to some registry as mentioned in #29. This also might work if you install the provider locally has described in https://github.com/nbering/terraform-provider-ansible/issues/29#issuecomment-674293354.

**Comment by nbering** [comment]
Assuming this is closed by #29.
